src/mount-support.c

```diff
@@ -293,3 +294,55 @@ void setup_slave_mount_namespace()
 		die("can not make make / rslave");
 	}
 }
+
+void setup_bind_mounts(const char *appname)
+{
+	debug("%s: %s", __FUNCTION__, appname);
+
+	FILE *f = NULL;
+	const char *bind_profile_dir = "/var/lib/snapd/bind/profiles/";
+
+	char profile_path[PATH_MAX];
+	int snprintf_rc =
+	    snprintf(profile_path, sizeof(profile_path), "%s/%s.bind",
+		     bind_profile_dir, appname);
```

Comment:
Please switch to the helper function must_snprintf() so that you don't have to do your own error handling here.